modules/proc-rhel-installing-jdk-rpm.adoc

```diff
@@ -18,3 +21,7 @@ $ javac -version
 javac 11.0.3
 ----
 
+[NOTE]
+====
+Depending on the default JDK selection priority for your {rhel} system via `alternatives`, either version 8 or version 11 will be default. See section 3, "Configuring OpenJDK for {rhel}", for details.
```

Comment:
We shouldn't mention a specific section number because this is a module and could be included in documentation anywhere, instead we should just link to the section's ID tag.

Reply:
Thanks. Should be fixed now.